markdown/3.0/en/documentation/include-scripts.md

```diff
@@ -4,19 +4,15 @@
 
 The following section shows a boilerplate for including FACT-Finder Web Components in your website. `custom-elements-es5-adapter.js` and `webcomponents-loader.js` will load the required Polyfills minimized to the needs of the calling browser. 
 
+Note that the [default-styles.css](todo-link-to-github-after-release) within our dist folder is already bundled into `bundle.js` and hence is not necessary to extra load it into your page. It is there only for reference.
```

Comment:
Please don't forget it though ;) 
Maybe create a ticket now?

Reply:
[This](https://github.com/FACT-Finder-Web-Components/ff-web-components/blob/release/3.0/dist/default-styles.css) is it from the release project.